packages/features/bookings/Booker/components/BookEventForm/BookEventForm.tsx

```diff
@@ -84,6 +84,11 @@ export const BookEventForm = ({
     return eventType?.price > 0 && !Number.isNaN(paymentAppData.price) && paymentAppData.price > 0;
   }, [eventType]);
 
+  const paymentCurrency = useMemo(() => {
+    if (!eventType) return "USD";
+    return getPaymentAppData(eventType)?.currency || "USD";
```

Comment:
Is this `currency` type string? It makes more sense for `currency` to be an enum

Reply:
Yes it is string, using already existing function code from https://github.com/calcom/cal.com/blob/880fe151c079b38ce12d8828fcb8537004a22df8/packages/lib/getPaymentAppData.ts#L37